Production/test/condorSub/step2.sh

```diff
@@ -57,14 +57,21 @@ if [[ $CMSEXIT -ne 0 ]]; then
 fi
 
 # copy output to eos
-echo "xrdcp output for condor"
+echo "xrdcp/gfal-copy output for condor"
 for FILE in *.root; do
-	echo "xrdcp -f ${FILE} ${OUTDIR}/${FILE}"
-	stageOut -x "-f" -i ${FILE} -o ${OUTDIR}/${FILE}
+	if [[ ( "$CMSSITE" == "T1_US_FNAL" && "$USER" == "cmsgli" && "${OUTDIR}" == *"root://cmseos.fnal.gov/"* ) ]]; then
```

Comment:
this should happen outside of the loop (then the message above can be specific)

Reply:
This should be fixed now.